src/styles/main.scss

```diff
@@ -1,3 +1,217 @@
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+
 body {
   margin: 0;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+
+  font-weight: 500;
+  font-size: 10px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo-image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link.is-active {
+  color: $main-color;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition: color 0.3s;
+  color: $text-accent-color;
+
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+  background-color: $main-color;
+}
+
+.nav__link:hover {
+  color: $main-color;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
```

Comment:
fine solution, but i am not sure about it, because of  [CHECKLIST](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)'s
```[LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards```